glassfish-tck-runner/pom.template.xml

```diff
@@ -361,7 +361,7 @@
                 <dependency>
                     <groupId>fish.payara.arquillian</groupId>
                     <artifactId>arquillian-payara-server-remote</artifactId>
-                    <version>3.0.alpha1</version>
+                    <version>3.0.alpha3</version>
```

Comment:
Why not alpha 5?

Reply:
Ah, because it causes the tests to freak out - interesting...

Reply:
This should be fixed in Patrik's PR https://github.com/payara/ecosystem-arquillian-connectors/pull/187 - it just needs to be merged and released

Reply:
The issue I was when trying alpha 5 was that for some reason something is still maintaining the Servlet 5.0 Protocol and this connector version is using the Servlet 6.0 protocol. This is causing Nullpointer exception for most of the tests